src/utils/settingsSync.ts

```diff
@@ -49,7 +49,8 @@ export async function exportSettings({ minify }: { minify?: boolean; } = {}) {
 }
 
 export async function downloadSettingsBackup() {
-    const filename = "vencord-settings-backup.json";
+    // The date is formatted as yyyy-mm-dd.
+    const filename = `vencord-settings-backup-${(new Date).toLocaleDateString('ky')}.json`;
```

Comment:
using ky locale here is a bit strange and based on my testing this seems to format with slashes which will cause issues on non windows

moment can easily do custom format
```suggestion
    const filename = `vencord-settings-backup-${moment().format("YYYY-MM-DD")}.json`;
```

Reply:
> using ky locale here is a bit strange and based on my testing this seems to format with slashes which will cause issues on non windows

Huh, that's weird, since it used dashes on my system (Linux). Maybe it's different on different operating systems, which I did not expect.

Reply:
i don't think os matters, it's just js :p 

it might be that it automatically changed / -> - cause / isnt allowed in filenames?

![image](https://github.com/Vendicated/Vencord/assets/45497981/39a8094e-aed1-48b3-97a8-b3ce82c130e7)

either way doesnt matter, i already fixed & pushed it. thanks again